### PR TITLE
[DO NOT MERGE EVER] The Honkbot!

### PR DESCRIPTION
**Pull request by GunHog**
Someone asked me for the Honkbot code I made from pure boredom and desire to better understand bot code.
<b>THIS IS NOT INTENDED TO BE ADDED TO A LIVE SERVER. NOT NOW, NOT LATER, NOT EVER.</b>
This bot is designed to make you angry. It is designed to annoy people. It is designed to be horrible in every way.

It is not balanced. It is not coded well. It is barely functional. It does not meet coding standards. It is a toy for bored coders, that is all.

All who download the code are free to do with it what they will, of course, but I assume no responsibility for the maintainability or grievances as a result of this bot's deployment. For clarification, I repeat,

<b>DO NOT MERGE THIS, EVER.</b>

### CODE REVIEW

File: code/game/machinery/bots/honkbot.dm [hunk 1]
```diff
@@ -0,0 +1,418 @@
...
+		"What do you call a lizard-man detective? An investiGATOR!",
+		"Did you here about the guy who lost his whole left side? He�s all-right now!",
+		"What do you call a cow with no legs? GROUND BEEF!",
+		"What do you call cheese that isn't yours? NACHO CHEESE!",
+		"OH GOD ITS FREE CALL THE SHUTTLE!")
+
+	else //Dirty or inapproprate jokes here. Keep it IC and legal please!
+		JL = list("Yo mama so hairy, When you were born, You got carpet burn!",
+		"What goes in dry and comes out wet? A tea bag!",
+		"Did you hear about the kidnapping at the Dorms? It�s ok, she woke up!",
```
Comment (RemieRichards) [inline code/game/machinery/bots/honkbot.dm:375]: this one's not really Diry or inappropriate. it's a kid sleeping.

File: code/game/machinery/bots/honkbot.dm [hunk 2]
```diff
@@ -0,0 +1,418 @@
...
+	icon = 'icons/obj/aibots.dmi'
+	icon_state = "honkbot0"
+	layer = 5.0
+	density = 0
+	anchored = 0
+	health = 40
+	maxhealth = 40
+	fire_dam_coeff = 0.5
+	brute_dam_coeff = 0.2 //Clowns are natural punching bags!
+//	weight = 1.0E7
```
Comment (RemieRichards) [inline code/game/machinery/bots/honkbot.dm:17]: Very heavy.

Comment (GunHog) [inline code/game/machinery/bots/honkbot.dm:17]: ...WHY are you reviewing this? 

<B>Remie: JUST AS I CAN EDIT YOUR MESSAGES </B>

Comment (RemieRichards) [inline code/game/machinery/bots/honkbot.dm:17]: Because I can.

**Comment by ghost** [comment]
Merge it.

**Comment by RemieRichards** [comment]
@Nienhaus I would, but the code is actually pretty pantsu.

**Comment by ghost** [comment]
Shhhh. No one needs to know.

**Comment by RemieRichards** [comment]
@Nienhaus I'll know! ;_;

**Comment by ghost** [comment]
Let me merge it then.

**Comment by RemieRichards** [comment]
non

**Comment by ghost** [comment]
Pretty please?

**Comment by RemieRichards** [comment]
non.

File: code/game/machinery/bots/honkbot.dm [hunk 3]
```diff
@@ -0,0 +1,418 @@
...
+/obj/machinery/bot/honkbot/proc/tell_joke() //Because clowns are supposed to be funny!
+	var/joke
+	var/list/JL //The list of jokes.
+	if(emagged != 2) //"Clean" joke list!
+		JL = list( "What did the wall say to the other wall? MEET YOU AT THE CORNER!",
+		"Why are AIs so calm? BECAUSE NOTHING GETS UNDER THEIR SKIN!",
+		"I heard that being a vampire really SUCKS!",
+		"Slime-people can't lie to me. I CAN SEE RIGHT THROUGH THEM!",
+		"Why do cyborgs hate pie? BECAUSE THEY DON'T HAVE THE STOMACH FOR IT!",
+		"Who as the mose famous skeleton detective? SHERLOCK BONES!",
```
Comment (ike709) [inline code/game/machinery/bots/honkbot.dm:363]: is\* most*

**Comment by Iamgoofball** [comment]
merge like sanic

**Comment by ike709** [comment]
merge now

**Comment by ghost** [comment]
> merge now

**Comment by GunHog** [comment]
> > merge now

https://github.com/tgstation/tgstation/blob/6de334414e7c15a92b7bb1a4dc000004d13e5ead/code/modules/mob/living/simple_animal/bot/honkbot.dm This is actually a thing on /tg/  It is not MY code, but I giggle at seeing it anyway

**Pull request closed without merging**